src/Servers/HttpSys/src/RequestProcessing/RequestContext.FeatureCollection.cs

```diff
@@ -602,6 +610,9 @@ IHeaderDictionary IHttpResponseTrailersFeature.Trailers
 
         public bool CanDelegate => Request.CanDelegate;
 
+        // Should therer be a cts for this so it could get triggered when RequestClose called?
+        CancellationToken IConnectionLifetimeNotificationFeature.ConnectionClosedRequested { get; set; }
```

Comment:
This can return a default non-cancellable token for now. We can revisit if there's interest in the token.